app/config.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+
+STATIC_ABS_PATH = os.path.abspath("static")
```

Comment:
This should be in app/config.py.example

(Please `git rm` the `app/config.py` file)

Reply:
Great job! Just fix this one and we're good to go.

Reply:
done